hoot-core/src/main/cpp/hoot/core/conflate/highway/Roundabout.cpp

```diff
@@ -259,7 +267,8 @@ void Roundabout::removeRoundabout(boost::shared_ptr<OsmMap> pMap)
   }
 
   // Find our center coord...
-  _pCenterNode = getNewCenter(pMap);
+  if (!_pCenterNode)
+    _pCenterNode = getNewCenter(pMap);
```

Comment:
In some cases the centerpoint was created twice and the first one was leaked into the output dataset.  Here we just use it if it already exists.